mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -47,6 +47,12 @@ enum class ReinterpretMapScope {
   kExceptGeneric, // reinterprets operation other than linalg.generic
 };
 
+/// Defines a scope for reinterpret map pass.
+enum class DebugSparseIteration {
+  kNone,          // generate fully inlined (and functional) sparse iteration
```

Comment:
I would not call this "none"
How about

kFunctional or kComplete, or something like that